### DISCUSSION TITLE
Reorder items by drag and drop

### DISCUSSION THREAD

**Discussion by tesar-tech** [Megabit/Blazorise#2949] [Ideas]
I am looking for this feature:
https://user-images.githubusercontent.com/33880579/134155084-3d235248-7fa0-4a9a-a754-de517880b8fe.mp4

Basically list with ability to reorder items.

- [Question on SO.](https://stackoverflow.com/questions/59302823/blazor-re-ordering-a-list-with-drag-and-drop)
  - with demo presented in video: https://blazorfiddle.com/s/8jurefka
- [Drag and Drop demo for Blazor](https://chrissainty.com/investigating-drag-and-drop-with-blazor/) (Chris Sainty)

 Are there intentions to make this feature? Would you be interested in merging my (future) solution?

**Comment by stsrki** [comment]
Well, we already have an open ticket for drag and drop support #2633. So yes, we plan to implement it in the near future. The plan is to have it as generic as possible so that it can be used for different scenarios. As you can see in one of my previous comments, the plan is to have Dropzone and Draggable components that would be wrappers that could hold different Blazorise components and HTML elements.

**Comment by tesar-tech** [comment]
For anyone interested or for an inspiration: https://github.com/tesar-tech/DragAndDropList
